packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -11987,13 +11994,18 @@ void main() {
           children: <Widget>[
             SizedBox(key: key, width: 200, height: 200),
             Actions(
-              actions: <Type, Action<Intent>>{EditableTextTapUpOutsideIntent: overrideAction},
+              actions: <Type, Action<Intent>>{
+                // The tap down has to be overridden so that the tap up will register.
```

Comment:
Is this the expected behavior? I would expect that I can override `EditableTextTapUpOutsideIntent` without overriding `EditableTextTapOutsideIntent`.

Reply:
The default behavior of `EditableTextTapOutsideIntent` is to unfocus the text field (except on mobile touch events). That means that the text field would be unfocused on the tap up event, blocking `EditableTextTapUpOutsideIntent`.
But I see how that might not be intuitive. I see a few solutions:
* Document the current behavior better
* Remove the focus check from `onTapUpOutside` (Close this PR)
* Check if the text field had focus previously

I'm open to other solutions as well

Reply:
> Remove the focus check from onTapUpOutside (Close this PR)
If we keep the focus check, then `onTapOutside` will never fire unless `onTapOutside` is overridden. If we remove it, this means every `TextField` that doesn't have focus and was "tapped outside" will receive a "onTapUpOutside", this doesn't seem like the correct behavior. I think it makes sense that only the `TextField` that was focused/previously focused receives "tapped outside" events.

> Check if the text field had focus previously
This sounds reasonable to me. The focus should probably be checked in `onTapOutside`, before we unfocus.